README.md

```diff
@@ -278,6 +278,8 @@ expect(double).to receive(:msg).with(1, duck_type(:abs, :div), "b") #2nd argumen
 expect(double).to receive(:msg).with(hash_including(:a => 5)) # first arg is a hash with a: 5 as one of the key-values
 expect(double).to receive(:msg).with(array_including(5)) # first arg is an array with 5 as one of the key-values
 expect(double).to receive(:msg).with(hash_excluding(:a => 5)) # first arg is a hash without a: 5 as one of the key-values
+expect(double).to receive(:msg).with(start_with('a')) # any matcher, custom or from rspec-expectations
+expect(double).to receive(:msg).with(satisfy{|data| data.dig(:a, :b, :c) == 5 }) # any expectation
```

Comment:
I don't quite understand why the comment here says "expectation", while this is also an example of using a matcher as an argument matcher for `receive`. Is `satisfy` somehow different from anything above?

Reply:
> Is satisfy somehow different from anything above?

I think that technically, from a library perspective, you're right that there isn't much difference between `satisfy` and the other examples.

But for a RSpec user, I think it is nice that the documentation points out that `satisfy` allows you to assert anything you want.